ome/datasource_baseline_compliance_report.go

```diff
@@ -156,32 +160,36 @@ func (g configurationReportDataSource) Read(ctx context.Context, req tfsdk.ReadD
 			)
 			return
 		}
-		state.BaseLineName.Value = config.BaseLineName.Value
-		state.ID.Value = config.BaseLineName.Value
+		if !config.BaseLineName.IsUnknown() {
+			state.BaseLineName = config.BaseLineName
+		}
+		if !config.ID.IsUnknown() {
+			state.ID = config.BaseLineName
+		}
```

Comment:
Can u pls explain why we are doing the check for unknowns? There should not be any unknowns at the Read stage.
Also, dont we need "else" blocks for these "if"s?

Reply:
unsetting the unknown after testing the datasource and resource